YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -688,6 +703,11 @@ bool Repository::ask_and_set_git_config_entry(const std::string& config_entry, c
     return true;
 }
 
+bool Repository::is_file_tracked(const std::string& path)
+{
+    return !repo_.get_untracked_objects_in_path(path).empty();
```

Comment:
does it still work when path is in .gitignore ?

Reply:
Just tested, and it works.